lib/src/services/pub_service.dart

```diff
@@ -21,7 +22,7 @@ class PubService {
     for (var package in packages) {
       if (!package.contains(ksStackedCli)) continue;
 
-      version = package.split(' ').last;
+      version = package.split(' ').elementAt(1);
```

Comment:
Why do we need to change this? 🤔

Reply:
It's fixing this issue: https://github.com/Stacked-Org/stacked/issues/979 But If you want I can do a separate PR. I just needed it to actually test my PR, because I was not able to register the local package in pub otherwise.

Reply:
I removed it so we can finish this PR.